functions/reportToIWF.ts

```diff
@@ -75,7 +89,25 @@ export const handler: ServerlessFunctionSignature<EnvVars, Event> = TokenValidat
 
       const channelID = context.IWF_API_CHANNEL_ID ? parseInt(context.IWF_API_CHANNEL_ID, 10) : 51;
 
-      const body: Required<Pick<IWFReportPayload, 'Reporting_Type' | 'Live_Report' | 'Media_Type_ID' | 'Report_Channel_ID' | 'Origin_ID' | 'Submission_Type_ID' | 'Reported_Category_ID' | 'Reported_URL' | 'Reporter_Anonymous' | 'Reporter_First_Name' | 'Reporter_Last_Name' | 'Reporter_Email_ID' | 'Reporter_Country_ID' | 'Reporter_Description'>> = {
+      const body: Required<
+        Pick<
+          IWFReportPayload,
+          | 'Reporting_Type'
+          | 'Live_Report'
+          | 'Media_Type_ID'
+          | 'Report_Channel_ID'
+          | 'Origin_ID'
+          | 'Submission_Type_ID'
+          | 'Reported_Category_ID'
+          | 'Reported_URL'
+          | 'Reporter_Anonymous'
+          | 'Reporter_First_Name'
+          | 'Reporter_Last_Name'
+          | 'Reporter_Email_ID'
+          | 'Reporter_Country_ID'
+          | 'Reporter_Description'
+        >
+      > = {
```

Comment:
I don't think you need to define the type again here, since this is picking all properties from `IWFReportPayload`.

`body: IWFReportPayload = { .... `